tracing-subscriber/src/filter/mod.rs

```diff
@@ -1,13 +1,68 @@
-//! [`Subscriber`]s that control which spans and events are enabled by the wrapped
-//! subscriber.
+//! [Subscribers](crate::subscribe) that control which spans and events are
+//! enabled by the wrapped collector.
 //!
-//! [`Subscriber`]: crate::fmt::Subscriber
+//! This module contains a number of types that provide implementations of
+//! various strategies for filtering which spans and events are enabled. For
+//! details on filtering spans and events using [`Subscribe`] implementations,
+//! see the [`subscribe` module documentation].
+//!
+//! [`subscribe` module documentation]: crate::subscribe#filtering-with-subscribers
+//! [`Subscribe`]: crate::subscribe
+mod filter_fn;
 mod level;
 
-pub use self::level::{LevelFilter, ParseError as LevelParseError};
-
 feature! {
     #![all(feature = "env-filter", feature = "std")]
     mod env;
     pub use self::env::*;
 }
+
+feature! {
+    #![all(feature = "registry", feature = "std")]
+    mod subscriber_filters;
+    pub use self::subscriber_filters::*;
+}
+
+pub use self::filter_fn::*;
+#[cfg(not(feature = "registry"))]
+pub(crate) use self::has_plf_stubs::*;
+
+pub use self::level::{LevelFilter, ParseError as LevelParseError};
+
+#[cfg(not(all(feature = "registry", feature = "std")))]
+pub(crate) use self::has_plf_stubs::*;
+
+feature! {
+    #![any(feature = "std", feature = "alloc")]
+    pub mod targets;
+    pub use self::targets::Targets;
+
+    mod directive;
+    pub use self::directive::ParseError;
+}
+
+/// Stub implementations of the per-layer-fitler detection functions for when the
```

Comment:
note that there's a spelling mistake in "fitler":

```suggestion
/// Stub implementations of the per-subscriber-filter detection functions for when the
```